src/Pages/Signup/Signup.scss

```
@@ -0,0 +1,285 @@
+@import '../../Styles/common.scss';
+@import url('https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css');
+.Signup-Main {
+  background-color: #fff;
+  width: 100%;
+  height: 1316px;
+  padding-bottom: 120px;
+}
+.Signup-Container {
+  width: 1050px;
+  height: 1316px;
+  background-color: #fff;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 155px;
+  margin-bottom: 419px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  h3 {
+    @extend %flexCenter;
```

```suggestion
  align-items: center;
  
  h3 {
    @extend %flexCenter;
```
모든 새로운 셀렉터는 한줄 띄워주세요

---

알겠습니다~